### PR TITLE
feat(coupon): improvement coupon ui component on template

### PR DESCRIPTION
**Pull request by muhammadanova**
To do :
- Improve style coupon popup full device (1 object style OrderSummaryBox & ListPaymentMethod)
- Remove isCouponAccordion (OrderSummaryBox & ListPaymentMethod)
- Add props function onSuccessCopyCodeCoupon (OrderSummaryBox & ListPaymentMethod)
- Add copy icons
- Add locales coupon
- Points popup as is
- Up version to 2.16.1 cause 2.16.4 have bug on ListPaymentMethod selected

SS pow : 

Screen Shot 2022-10-07 at 14 12 21
Screen Shot 2022-10-07 at 14 12 33
Screen Shot 2022-10-07 at 14 12 43
Screen Shot 2022-10-07 at 14 13 15
Screen Shot 2022-10-07 at 14 13 56
Screen Shot 2022-10-07 at 14 14 06
Screen Shot 2022-10-07 at 14 14 16
Screen Shot 2022-10-07 at 14 14 25
Screen Shot 2022-10-07 at 14 14 53
Screen Shot 2022-10-07 at 14 15 13
Screen Shot 2022-10-07 at 14 15 25
Screen Shot 2022-10-07 at 14 15 39
Screen Shot 2022-10-07 at 14 15 58
Screen Shot 2022-10-07 at 14 16 07
Screen Shot 2022-10-07 at 14 16 25
Screen Shot 2022-10-07 at 14 16 32

### CODE REVIEW

File: public/scss/components/PopUpVoucherCoupon.module.scss [hunk 1]
```diff
@@ -455,10 +545,179 @@
...
   @include flex(row, center, flex-start);
-  padding: 0 10px;
+  @include typographyBuilder(400, 12, 16);
+  color: $color_red_other;
+  margin: 0;
+
+  &::before
+  {
+    content: "";
+    width: 13px;
```
Comment (mayun19) [inline public/scss/components/PopUpVoucherCoupon.module.scss]: ini bisa pakai fixedSize mas

File: public/scss/components/PopUpVoucherCoupon.module.scss [hunk 2]
```diff
@@ -239,18 +278,64 @@
...
+
+  .voucherDetailEstimate
+  {
+    @include flex(row, center, space-between);
+
+    &::after
+    {
+      content: "";
+      display: block;
+      width: 14px;
```
Comment (mayun19) [inline public/scss/components/PopUpVoucherCoupon.module.scss]: ini bisa pakai fixedSize mas

File: public/scss/components/PopUpVoucherCoupon.module.scss [hunk 3]
```diff
@@ -239,18 +278,64 @@
...
 
+.voucherDetailViewDetails
+{
+  @extend .btn;
+  padding: 0;
+  border: none;
+  background-color: transparent;
+  color: $color_blue;
+  @include ExtraSmallReguler;
+  @include absolute(8px, 12px);
```
Comment (mayun19) [inline public/scss/components/PopUpVoucherCoupon.module.scss]: bisa ditaruh di bawahnya @extend mas
```suggestion
  @extend .btn
  @include ExtraSmallReguler;
  @include absolute(8px, 12px);
```

File: public/scss/components/PopUpVoucherCoupon.module.scss [hunk 4]
```diff
@@ -455,10 +545,179 @@
...
+    display: none;
+    opacity: 0;
+    visibility: hidden;
+  }
+}
+
+.voucherDetailPopUpBody
+{
+  @include fixed(auto, auto, 0);
+  z-index: 1;
```
Comment (mayun19) [inline public/scss/components/PopUpVoucherCoupon.module.scss:581]: z-index nya bisa ditaruh di bawahnya @include flex

**Review by mayun19 — CHANGES_REQUESTED** [review]

**Comment by inezamandha** [comment]
mas aku mau nanya saja
kalau dia kosong enggak ada logonya memang apakah seperti itu (ada background abu abunya tanpa logo)?
image

atau seharusnya seperti punya anggara?
image

**Review by inezamandha — COMMENTED** [review]
pertanyaannya ada di atas ya hehe terkait desainnya

**Review by mayun19 — APPROVED** [review]

File: pages/[lng]/payment_method/index.tsx [hunk 5]
```diff
@@ -129,26 +129,27 @@ const PaymentMethods: FC<any> = ({
   const linksBreadcrumb = [`${i18n.t("header.home")}`, i18n.t("placeOrder.checkOrder")]
```
Comment (falmesino) [inline pages/[lng]/payment_method/index.tsx]:
```suggestion
  const linksBreadcrumb = [i18n.t("header.home"), i18n.t("placeOrder.checkOrder")]
```

giniin boleh pacc? gatel saia liatnya

File: public/scss/components/PopUpVoucherCoupon.module.scss [hunk 6]
```diff
@@ -159,6 +133,70 @@
...
+  margin-top: 12px;
+
+  input[class*="form-control"]
+  {
+    @include transition();
+    @include typographyBuilder(400, 16, 16);
+    width: 231px;
+    max-width: 100%;
+    height: 100%;
+    max-height: 100%;
```
Comment (falmesino) [inline public/scss/components/PopUpVoucherCoupon.module.scss]:
```suggestion
    @include fixedHeight(100%);
```

udah ada mixin ini kan ya kalo urban

File: public/scss/components/PopUpVoucherCoupon.module.scss [hunk 7]
```diff
@@ -455,10 +536,174 @@
...
+    color: $color_blue;
+  }
+}
+
+.voucherDetailPopUpContainer
+{
+  @extend .popup_overlay;
+  @include flex(row, center, center);
+  @include fixedWidth(100%);
+  @include fixedHeight(100%);
```
Comment (falmesino) [inline public/scss/components/PopUpVoucherCoupon.module.scss]:
```suggestion
  @include fixedSize(100%);
```

pake mixin ini saja ngab

**Review by falmesino — CHANGES_REQUESTED** [review]

**Review by falmesino — CHANGES_REQUESTED** [review]
request changes perintilan

**Review by inezamandha — APPROVED** [review]

**Review by falmesino — APPROVED** [review]

**Pull request merged**